server/src/main/java/com/projects/pms/empmapper/MapperImp.java

```diff
@@ -0,0 +1,31 @@
+package com.projects.pms.empmapper;
+
+import com.projects.pms.dtos.*;
+import com.projects.pms.models.Deductions;
+import com.projects.pms.models.Earnings;
+import com.projects.pms.models.Employee;
+import org.mapstruct.Mapper;
+import org.mapstruct.factory.Mappers;
+import org.springframework.stereotype.Component;
+
+@Component
+@Mapper(componentModel = "spring")
+public interface MapperImp {
+
+    MapperImp INSTANCE = Mappers.getMapper(MapperImp.class);
```

Comment:
@SunilSunny698 Why are you getting an instance of mappper.

Reply:
As we are autowiring the MapperImp, there is no need of this INSTANCE, so we are removing it now..